### ISSUE TITLE
How to use this software?

### ISSUE THREAD

**Issue by cthoyt** [markmartorilopez/OChemR#2]
Building on #1, a good README should have step-by-step instructions for a minimal example on how to use the software

**Comment by markmartorilopez** [comment]
Yes, as mentioned in the README, the repository was being updated to provide step-by-step instructions.

I recommend running the "End-to-End OChemR (From image to JSON)" section after installing the requirements and downloading the DETR weights and placing them in the correct folder.

Also, no OCR-trained weights are yet publicly provided. This means that the OCR will not show the same results as stated in the README since it will not endow the correct weights. 

Everything will be updated soon :)
